src/deroff.rs

```diff
@@ -1145,25 +1145,21 @@ fn deroff_files(files: &[String]) -> io::Result<()> {
             let mut decoder = GzDecoder::new(file);
             decoder.read_to_string(&mut string)?;
         } else {
-            match file.read_to_string(&mut string) {
-                Err(e) if e.kind() == io::ErrorKind::InvalidData => {
-                    // not valid UTF-8
-                    // TODO: This is a _bad_ workaround for latin1, we need to correctly decode input files
-                    let mut bytes = Vec::new();
-                    file.read_to_end(&mut bytes)?;
-
-                    for byte in bytes {
-                        string.push(byte as char);
-                    }
-                }
+            let mut bytes = Vec::new();
+            file.read_to_end(&mut bytes)?;
+
+            match String::from_utf8(bytes.clone()) {
```

Comment:
I don't think cloning here is a good idea, I believe that broken encodings are kinda a rare case nowadays. I personally think ignoring broken utf-8 files is the better choice or maybe just use `String::from_utf8_lossy` to replace any broken utf-8 characters.

Reply:
Maybe post-mvp?

Reply:
Yes, this could be post-mvp but I think it still needs not to do unnecessary cloning that is very likely useless.